src/globus_sdk/_testing/data/transfer/_common.py

```diff
@@ -1,5 +1,10 @@
 import uuid
 
-SUBMISSION_ID = str(uuid.UUID(int=2020))
-ENDPOINT_ID = str(uuid.UUID(int=1234))
-TASK_ID = str(uuid.UUID(int=1001))
+
+def _as_uuid(s: str) -> str:
+    return str(uuid.UUID(int=int(s, 36)))
```

Comment:
:heart:

Reply:
I learned this base 36 usage trick from someone. 😉